content/chapters/io/lecture/slides/intro.md

```diff
@@ -0,0 +1,34 @@
+## Backstory
+
+---
+
+### Backstory
+
+- You finish your summer internship
+- You explain the cool internal parts of the project to your friends
+- They hit you with: "How do I interact with it?"
+- You describe the data gathering algorithms again and see them lose interest
+
+What was missing?
+
+----
+
+#### Interactive Software is Good Software
+
+![interactive software](./media/interactive-software.png)
+
+----
+
+#### IO - The Need
+
+- Software does computation on data and returns data
+- The more data we provide to our application, the more relevant its result becomes
+- Interactivity
+
+----
+
+#### What We Want
+
+- Means for our application to communicate
+- Compatibility with various devices
+- Responsiveness (Non-blocking I/O)
```

Comment:
We also want "Performance" - fast I/O interfaces (fast transfer, I/O multiplexing)